src/AerialShip/SamlSPBundle/Resources/config/mongodb.yml

```diff
@@ -0,0 +1,17 @@
+
+parameters:
+    aerial_ship_saml_sp.state.store.sso.class: AerialShip\SamlSPBundle\Doctrine\SSOStateStore
+    aerial_ship_saml_sp.backend_type_mongodb: 1
+
+services:
+    # factory will be set in bundle extension, listed here to be used as a reference in other services
+    aerial_ship_saml_sp.document_manager:
+        class: Doctrine\ODM\MongoDB\DocumentManager
+        arguments:
+            - %aerial_ship_saml_sp.model_manager_name% # not realy used since factory is set in extension
```

Comment:
typo: ``really``.

and the argument is used: it is the argument passed to the factory method